src/features/DashboardPage/index.jsx

```diff
@@ -40,6 +40,28 @@ const DashboardPage = () => {
     fetchExams();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const handleCancelExam = async (vueAppointmentId) => {
+    setExams((prev) => prev.map(
+      (exam) => (exam.vue_appointment_id === vueAppointmentId ? { ...exam, loading: true } : exam),
+    ));
+
+    try {
+      const response = await cancelExam(vueAppointmentId);
+
+      if (response?.data?.url) {
```

Comment:
Just a question, this is the same structure response for the actions? to adjust my PR with that